messages/3_anoncred_proof_request.json

```diff
@@ -1,9 +1,9 @@
 {
     "nonce": "22456608037449564223",
-    "name": "NextcloudCombinedRequest",
+    "name": "NextcloudCredentialAC",
     "version": "1.0",
     "requested_attributes": {
-        "NextcloudCredentialAC": {
+        "ref2": {
```

Comment:
der input_descriptor hat immer noch die ID NextcloudCredentialAC

Reply:
ah, verstehe, ref2 verwendest Du nur in der submission. Bedeutet, dass ref2 durch das Wallet vorgegeben wird, nicht den Verifier. Korrekt?

Reply:
Ja genau. Ich denke dadurch, dass das Wallet in der Presentation Submission den Pfad angibt kann der Verifier den Wert hier frei wählen.